README.md

```diff
@@ -1 +1,17 @@
 # Tic-Tac-Toe
+
+## Application Quick-start Commands
+
+* To play game:
+
+  * `cd` into `App` dir
+
+  * run app w/ `dotnet run` 
+
+## Dependencies
+
+* `moq` - most popular and friendly mocking library for .NET
```

Comment:
i think there's some dependencies missing here, probably need the right cmd line library to be able to run `dotnet run`